src/hotspot/share/asm/codeBuffer.cpp

```diff
@@ -1009,11 +1009,12 @@ void CodeBuffer::log_section_sizes(const char* name) {
   }
 }
 
-void CodeBuffer::finalize_stubs() {
+bool CodeBuffer::finalize_stubs() {
```

Comment:
It's not actually relevant to this fix but it might be worth making a small change here. There is actually no need to call `pd_finalize_stubs` if `_finalize_stubs` is already false. So, you could return straight away if it is already false.

Reply:
Thanks for the quick review, Andrew. Yes you are right. Tested a simple AArch64 tier1 and some tests for zero.

For backends excluding x86, aarch64 and riscv, `CodeBuffer::supports_shared_stubs()` are all false and there are trampolines, so `_finalize_stubs` is always a false value.